projects/packages/sync/src/class-actions.php

```diff
@@ -675,6 +675,36 @@ public static function initialize_woocommerce() {
 		add_filter( 'jetpack_sync_modules', array( __CLASS__, 'add_woocommerce_sync_module' ) );
 	}
 
+	/**
+	 * Initializes sync for Instant Search.
+	 *
+	 * @access public
+	 * @static
+	 */
+	public static function initialize_search() {
+		if ( false === class_exists( 'Automattic\\Jetpack\\Search\\Module_Control' ) ) {
+			return;
+		}
+		$search_module = new \Automattic\Jetpack\Search\Module_Control();
```

Comment:
Quick question: if the Sync package relies on that class and method, do we need to add the Search package as a new dependency of the Sync package, or do we only rely on the check above?

Reply:
We can't have Sync dependent on Search. Long term this code will move to the Search package. 

We only want to modify the allow lists if the site has an active Instant Search plan which is why we are checking for the class definition to determine if any loading should occur.